src/list.hpp

```diff
@@ -106,6 +106,8 @@ class List {
         size_t valid_count;
     };
 
+    LinkViewRef get_linkview() const { return m_link_view; }
```

Comment:
@tgoyne I don't think this would be improved by exposing an `AnyThreadConfined` here instead since it has a `get_linkview()` property on itself anyway. Thoughts?